tests/test_semver.rs

```diff
@@ -1,4 +1,5 @@
-#![cfg(feature = "semver")]
+#![cfg(all(feature = "semver", not(feature = "semver-1")))]
```

Comment:
```suggestion
#![cfg(feature = "semver")]
```

I think this can stay as it was now